main.ts

```diff
@@ -72,6 +76,44 @@ class Translator {
     this.visit(expr);
   }
 
+  /** Searches for super() calls and emits an initializer for it if found. */
+  maybeEmitInitializerForSuperCall(body: ts.Block) {
```

Comment:
nit: naming: this is emitting the colon, and therefore the entire initializer list, which could include field initializers.